Wireframe/index.html

```diff
@@ -10,20 +10,70 @@
-      </article>
+      <div class="top">
+        <section class="article">
+          <article>
+            <img src="placeholder.svg" alt="" />
+<<<<<<< HEAD
+            <h2>What is the a placeholder?</h2>
+=======
+            <h2>What is a placeholder?</h2>
+>>>>>>> main
+            <p>
+              A placeholder in a website is a temporary element or text that reserves space for future content, such as images, text, or forms, during the design and development process. Placeholders, often generated by dummy text like "Lorem ipsum" or placeholder images, allow designers and developers to visualize the final layout, maintain design structure, and test the user experience without waiting for actual content to be ready. 
+            </p>
+            <a href="https://placehold.co/" target="_blank">Read more</a>
+          </article>
+        </section>
+      </div>
+
+      <div class="top">
+        <section class="article">
+          <article>
+            <img src="sitemap.jpg" alt="" />
+            <h2>What is a placeholder?</h2>
+            <p>
+              A placeholder in a website is a temporary element or text that reserves space for future content, such as images, text, or forms, during the design and development process. Placeholders, often generated by dummy text like "Lorem ipsum" or placeholder images, allow designers and developers to visualize the final layout, maintain design structure, and test the user experience without waiting for actual content to be ready. 
+            </p>
+            <a href="https://www.20i.com/blog/sitemaps-complete-guide/" target="_blank">Read more</a>
+          </article>
+        </section>
+      </div>
+
+      <div class="grid"> 
+        <section class="article"> 
+            <article>
+              <img src="wireframe.png" alt="Wireframe illustration"> 
+              <h2>What is the purpose of a wireframe?</h2> 
+              <p>A wireframe is a simple blueprint of a webpage or app. It focuses on structure and layout, showing where elements like images, text, and buttons will go before visual design begins.
+              </p> 
+              <a href="https://wireframe.cc/" target="_blank">Read More</a> 
+            </article>
+        </section> 
+      </div>
+     
+      <div class="branch">
+        <section class="article"> 
+          <article>
+            <img src="git-branch.png" alt="Git branch illustration"> 
+            <h2>What is a branch in Git?</h2> 
+            <p>A branch in Git allows developers to work on new features or fixes separately from the main codebase. This makes collaboration easier and ensures the main project stays stable while changes are tested.
+            </p> 
+            <a href="https://www.w3schools.com/git/git_branch.asp">Read More</a> 
+          </article>
+        </section> 
+      </div>
     </main>
+
     <footer>
       <p>
         This is the default, provided code and no changes have been made yet.
```

Comment:
Can you review the footer requirements for this exercise? It says:
 "The page footer is fixed to the bottom of the viewport."